exploration/maintaining-registry.md

```diff
@@ -155,73 +169,80 @@ _What prior decisions and existing conditions limit the possible design?_
 
 _Describe the proposed solution. Consider syntax, formatting, errors, registry, tooling, interchange._
 
-The MessageFormat WG will maintain three separate function and option registries 
-beginning with the LDML46 release.
-Future updates to these registries will coincide with LDML releases.
-
-Each registry consists of a set of template-derived documents.
-Each _function_ or _option_ entry in a registry consists of a separate document.
-An _option_ entry in a given registry will not be created if there is a corresponding
-_function_ entry in the same registry for which it is an _option_.
-Proposals to include functions into the default registry
-or to include functions or options into the RGI registry
-or to include functions or options into the Unicode reserved namespace registry
-need to follow a specific process.
-
-The three registries are:
-
-1. **Default Registry**
-This is a _function_ registry.
-It includes _functions_ (and only functions) that are normatively required to be
-implemented by all implementations.
-Each function describes it's operand or operands,
+The MessageFormat WG will a set of specifications
+that standardize the implementation of functions and options in the default namespace of 
+MessageFormat v2 beginning with the LDML46 release.
+Implementations and users are strongly discourages from defining values that use
+the default namespace.
+Future updates to these sets of functions and options will coincide with LDML releases.
+
+Each _function_ is described by a single specification document.
+Each such document will use a common template.
+A _function_ can be a _formatting function_,
+a _selector_,
+or both.
+
+The specification will indicate if the _formatting function_,
+the _selector function_, or, where applicable, both are `Standard` or `Optional`.
```

Comment:
We really should finally align on how to speak about resolved values, because that's clearly relevant here as well. Note also that _formatting function_ and _selector function_ are not terms that we've defined so far, and I don't think that they should be.